runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java

```diff
@@ -634,12 +637,13 @@ public void makeOverlayHint(PanelComponent panelComponent, ClueScrollPlugin plug
 			{
 				boolean equipmentFulfilled = requirement.fulfilledBy(equipment);
 				boolean inventoryFulfilled = requirement.fulfilledBy(inventory);
+				boolean combinedFulfilled = requirement.fulfilledBy(combined);
```

Comment:
probably run this only if neither equipment or inventory is fullfilled? to avoid unnecessary calls.